azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -529,6 +534,9 @@ public void shutdown() {
    */
   private void shutdownInternal() {
     getFlowRunnerManager().shutdown();
+    // Sleep for an hour to wait for web server updater thread
```

Comment:
Make it a Todo and add more details? Copying the description from the PR is fine.